README.md

```diff
@@ -19,18 +19,25 @@ All contributors are welcome! If you would like to contribute:
 2. `cd rides/`
 2. Create a python virtual environment, activate it and install requirements.
   - `virtualenv rides-venv`
+  - `cd ..`
   - `source rides-venv/bin/activate`
   - `pip install -r requirements.txt`
-5. You will need the _LDAP DN_, _LDAP PW_, _MySQL Databse_, and _OIDC Secret_. You can create your own if you are hosting your own application or ask me for these if you would like to contribute to CSH.
+5. You will need many credentials to run  _LDAP DN_, _LDAP PW_, _MySQL Databse_, and _OIDC Secret_. You can create your own if you are hosting your own application or ask me for these if you would like to contribute to CSH. I'd recommend making a config.sh to automatically do the exports you need. It is in the .gitignore so it will not go to the remote repository.
```

Comment:
```suggestion
5. You will need many credentials to run, see the exports below. You can create your own if you are hosting your own application or ask an rtps for these if you would like to contribute to CSH. I'd recommend making a config.sh to automatically do the exports you need. It is in the .gitignore so it will not go to the remote repository.
```
I don't think you need to specify all of this since it's below anyways